### ISSUE TITLE
Converting units in Velocity

### ISSUE THREAD

**Issue by k-choonkiat** [jupyter-widgets/ipyleaflet#603]
I would like to convert the units from m/s to Knots and have tried to do so using `Units` in Velocity but it didnt work

**Comment by davidbrochart** [comment]
Yes, lots of things seem to be broken in Velocity. Some traits are not synced with the front-end, including `units`. I tried to set `speedUnit` in `display_options` but it has no effect either... I'll look at it.

**Comment by davidbrochart** [comment]
Actually `units` should be taken into account through https://github.com/jupyter-widgets/ipyleaflet/blob/master/ipyleaflet/xarray_ds.py, but it's not.

**Comment by SchoolProjBase** [comment]
@davidbrochart thanks for your reply. Is there any update on this issue? I would like to tweak the units for velocity layer.

**Comment by davidbrochart** [comment]
Sorry, I didn't have time to work on it yet.

**Comment by SchoolProjBase** [comment]
Is there any hint or clue to resolve this issue?

**Comment by davidbrochart** [comment]
Sorry, I've been busy. Do you want to give it a try?
